src/inspector_js_api.cc

```diff
@@ -243,12 +243,12 @@ void Open(const FunctionCallbackInfo<Value>& args) {
 
   if (args.Length() > 0 && args[0]->IsUint32()) {
     uint32_t port = args[0].As<Uint32>()->Value();
-    agent->options()->host_port.port = port;
+    agent->host_port()->set_port(static_cast<int>(port));
```

Comment:
It is already possible to overflow before since we were assigning uint32_t to int anyways. I think we'd better turn `port` into a `uint16_t` (see the TODO in `node_options.h`) and do the checks whenever we set it, but I'd prefer to do that in another patch, because the port is actually settable from user land.